skcosmo/decomposition/_kernel_pcovr.py

```diff
@@ -160,47 +174,46 @@ class KernelPCovR(_BasePCA, LinearModel):
     >>> Y = np.array([[ 0, -5], [-1, 1], [1, -5], [-3, 2]])
     >>> Y = SFS(column_wise=True).fit_transform(Y)
     >>>
-    >>> kpcovr = KernelPCovR(mixing=0.1, n_components=2, kernel='rbf', gamma=2)
+    >>> kpcovr = KernelPCovR(mixing=0.1, n_components=2, regressor=KernelRidge(kernel='rbf', gamma=2), kernel='rbf', gamma=2)
     >>> kpcovr.fit(X, Y)
-        KernelPCovR(coef0=1, degree=3, fit_inverse_transform=False, gamma=0.01, kernel='rbf',
-           kernel_params=None, mixing=None, n_components=2, n_jobs=None,
-           alpha=None, tol=1e-12)
+        KernelPCovR(gamma=2, kernel='rbf', mixing=0.1, n_components=2,
+                    regressor=KernelRidge(gamma=2, kernel='rbf'))
     >>> T = kpcovr.transform(X)
-        [[ 1.01199065, -0.35439061],
-         [-0.68099591,  0.48912275],
-         [ 1.4677616 ,  0.13757037],
-         [-1.79874193, -0.27232032]]
+        [[-0.55119827, -0.21793572],
+         [ 0.3768726 ,  0.31208068],
+         [-0.76898956,  0.08511876],
+         [ 0.92488574, -0.18627707]]
     >>> Yp = kpcovr.predict(X)
-        [[-0.01044648, -0.84443158],
-         [-0.1758848 ,  0.16224503],
-         [ 0.1573037 , -0.84211944],
-         [-0.51133139,  0.32552881]]
+        [[ 0.51713163, -0.99453229],
+         [-0.16083953,  0.8378709 ],
+         [ 1.18143489, -1.01072628],
+         [-1.52011339,  1.13439986]]
     >>> kpcovr.score(X, Y)
-        (0.5312320029915978, 0.06254540655698511)
+        1.0000774522028972
```

Comment:
This seems bad -- the example is designed to be better than this

Reply:
The score output was  returning a tuple instead of the single value that is the current output and predates even this PR. Part of the issue might be the broken example (see comment above). I'll re-run with a different `gamma` and see what happens.

Reply:
Oh I don't have an issue with the single value -- it's more that a loss of 1.0 seems awful.

Reply:
I agree. I'll try re-doing this example with a different `gamma`